src/pss.rs

```diff
@@ -280,14 +280,12 @@ mod test {
             let digest = Sha1::digest(test[0].as_bytes()).to_vec();
             let sig = hex::decode(test[1]).unwrap();
 
-            let seed = SystemTime::now().duration_since(SystemTime::UNIX_EPOCH).unwrap();
+            let seed = SystemTime::now()
+                .duration_since(SystemTime::UNIX_EPOCH)
+                .unwrap();
```

Comment:
I don't quite like non-deterministic tests... Maybe we should port them to RNGs which use a fixed seed? Also it may be worth using `rand_core` instead of `rand`.

Reply:
I think this change was the result of my IDE applying rustfmt to the project, FWIW. That's kind of annoying, I was hoping it would work at the individual module level.

Reply:
rustfmt applied in #105

Reply:
created an issue to fix the seeds https://github.com/RustCrypto/RSA/issues/108